From: Carlos O'Donell <carlos_odonell@mentor.com>
Date: Mon, 7 May 2012 20:04:41 +0000 (-0400)
Subject: ARM: Define HAVE_ARM_PCS_VFP in config.h.
X-Git-Url: http://sourceware.org/git/?p=glibc.git;a=commitdiff_plain;h=6a43ec980c5a0500149ef37d4854eac0e270da6f;hp=05c2c9618f583ea4acd69b3fe5ae2a2922dd2ddc

ARM: Define HAVE_ARM_PCS_VFP in config.h.

If the compiler and flags would select the hard-float ABI
then the ARM configure fragment will set HAVE_ARM_PCS_VFP.
This is later used by the ARM shlib-versions to select
the appropriately named dynamic linker.
---

Upstream-Status: Backport
-Khem

Index: libc/config.h.in
===================================================================
--- libc.orig/config.h.in	2012-01-04 22:06:28.000000000 -0800
+++ libc/config.h.in	2012-05-08 11:25:56.581079069 -0700
@@ -233,4 +233,7 @@
 
 #define HAVE_REGEX 1
 
+/* The ARM hard-float ABI is being used.  */
+#undef HAVE_ARM_PCS_VFP
+
 #endif
